exocom-server/features/support/cli-world.ls

```diff
@@ -27,7 +27,8 @@ CliWorld = !->
 
   @create-mock-service-at-port = ({name, port, namespace}, done) ->
     (@service-mocks or= {})[name] = new MockService {port, name, namespace}
-    wait 200, done
+    @service-mocks[name].connect {}, ->
+      wait 200, done
```

Comment:
In the future we will just wait for the reply from Exocom. The service-mock will signal that it received that message, for example by emitting an event. This way we wait here exactly until the environment is set up, no matter how fast or how slow that is. 

Exorelays and services will also wait for that message before they signal that they are online.